dll/shellext/netshell/lanstatusui.cpp

```diff
@@ -354,6 +354,31 @@ InsertItemToListView(
     return -1;
 }
 
+BOOL tmToStr(
```

Comment:
should be static since it's an internal function

Reply:
ok